### PR TITLE
Feature/ui excel first row header

### PR DESCRIPTION
**Pull request by bdmogal**
Jiras:
[CDAP-11412](https://issues.cask.co/browse/CDAP-11412)
[CDAP-11822](https://issues.cask.co/browse/CDAP-11822)

### CODE REVIEW

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

**Comment by nitinmotgi** [comment]
LGTM 👍

File: cdap-ui/app/cdap/components/DataPrep/DataPrepTable/DataPrepTable.scss [hunk 1]
```diff
@@ -162,12 +157,6 @@ $col_type_font_color: gray;
 
         &.selected,
         &.dropdownOpened,
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepTable/DataPrepTable.scss]: these two should be deleted as well

Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepTable/DataPrepTable.scss]: These two should be removed as well

**Review by elfenheart — CHANGES_REQUESTED** [review]

File: cdap-ui/app/cdap/components/DataPrep/DataPrepTable/DataPrepTable.scss (hunk 1 above)
Comment (bdmogal) [reply cdap-ui/app/cdap/components/DataPrep/DataPrepTable/DataPrepTable.scss]: nice. removed.

**Review by ajainarayanan — COMMENTED** [review]
👍  Unless @elfenheart has any comments.

**Review by elfenheart — APPROVED** [review]

**Pull request merged**